src/Libraries/CoreNodeModels/Properties/Resources.resx

```diff
@@ -447,7 +447,7 @@
     <value>Starting accumulated value, to be passed into the first call to the Reductor function.</value>
   </data>
   <data name="ScopeIfDescription" xml:space="preserve">
-    <value>Scoped If statement</value>
+    <value>Returns the input of either the True or False input depending on what boolean value is toggled in the test input.</value>
```

Comment:
`Returns the result of either ... depending on the boolean value of the test input`

Reply:
addressed in bd2d1cb